src/Kunstmaan/NodeSearchBundle/Configuration/NodePagesConfiguration.php

```diff
@@ -334,7 +342,7 @@ protected function createDefaultSearchFieldsMapping(Index $index, $lang = 'en')
      * @param Index  $index
      * @param string $lang
      */
-    protected function setMapping(Index $index, $lang = 'en')
+    protected function setMapping(Index $index, $lang)
```

Comment:
Can you undo this change? I know it looks stupid but this is necessary to avoid BC breaks on older projects that still require $lang in this method.